components/layout/sidebar.js

```diff
@@ -71,18 +86,36 @@ export default function SideBar() {
           className="border-t-2 border-white bg-blue"
           popperArrow
         >
-          <MenuItem
-            icon={<FontAwesomeIcon icon={faGem} className="cursor-pointer" />}
-          >
-            Dashboard
+          <MenuItem>
```

Comment:
we should add an icon for Dashboard here, I don't think Tara knew we could have icons when the sidebar was minimized.  Just use `faHome` for now, it just looks odd to have `D...` when it's collapsed

Reply:
Yeah, I had an icon for the dashboard at first then noticed it wasn't in the design. I agree it looks betting in the collapsed view to have one. I'll put it back in.